sphinx-search-as-you-type.rst

```diff
@@ -0,0 +1,113 @@
+.. post:: July 15, 2019
+   :tags: gsoc, search, feature, in-doc-search
+   :author: Vaibhav
+   :location: LKO
+
+Search Update
+=============
+
+Giving users the ability to easily find the information that they
+are looking for has always been important for Read the Docs.
+This year, I, `Vaibhav Gupta`_, took the opportunity provided
+by Google Summer of Code to improve the search.
+The main goal of my `GSoC project`_ was to enable multiple hits per search result,
+improve the UI/UX around code search and to make an `Sphinx extension`_ to provide
+"search as you type" experience to the users.
+
```

Comment:
We should add another short paragraph here mentioning that it's live on our docs, and including a link so users can go try it out themselves.